faucet_injected.Dockerfile

```diff
@@ -11,19 +11,19 @@ ARG PROJECT_NAME=""
 
```

Comment:
I think this file can be renamed from `faucet_injected.Dockerfile` to simply `Dockerfile`.

Reply:
Agree, renamed.